tests/Responses/Expected/shipment/shipment_index_response.json

```diff
@@ -0,0 +1,188 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 1,
+    "total": 2,
+    "links": {
+        "self": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10",
+        "first": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10",
+        "last": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10",
+        "next": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10"
+    },
+    "items": [
+        {
+            "id": @integer@,
+            "state": "ready",
+            "method": {
+                "code": "UPS",
+                "name": "UPS",
+                "description": "@string@"
+            },
+            "order": {
+                "id": @integer@,
+                "number": "000000001",
+                "channel": "US_WEB",
+                "currencyCode": "USD",
+                "localeCode": "en_US",
+                "state": "new",
+                "checkoutState": "completed",
+                "checkoutCompletedAt": "@string@.isDateTime()",
+                "paymentState": "authorized",
+                "payments": [
+                    {
+                        "state": "authorized",
+                        "method": {
+                            "code": "cash_on_delivery",
+                            "name": "Cash on delivery",
+                            "description": "@string@"
+                        }
+                    }
+                ],
+                "shippingState": "ready",
+                "shippingAddress": {
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "countryCode": "UA",
+                    "street": "Main street",
+                    "city": "Odesa",
+                    "postcode": "65000",
+                    "phoneNumber": "380951234567"
+                },
+                "billingAddress": {
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "countryCode": "UA",
+                    "street": "Main street",
+                    "city": "Odesa",
+                    "postcode": "65000",
+                    "phoneNumber": "380951234567"
+                },
+                "items": [
```

Comment:
The items should be removed from there and added to the root element. And it should only be the items present on this particular shipment